include/deal.II/non_matching/mapping_info.h

```diff
@@ -402,7 +402,7 @@ namespace NonMatching
     std::vector<Quadrature<dim>> quadrature_vector(n_cells);
     for (unsigned int cell_index = 0; cell_index < n_cells; ++cell_index)
       quadrature_vector[cell_index] =
-        Quadrature<dim>(quadrature_vector[cell_index].get_points());
+        Quadrature<dim>(unit_points_vector[cell_index]);
```

Comment:
@fdrmrc Great you spotted this! Could you add a test!

FYI @fdrmrc

Reply:
Nice self-reference in the original code :-)